### PR TITLE
Add nodebook

### PR DESCRIPTION
**Pull request by cshaley**
Add nodebook, repeatable analysis plugin for Jupyter notebook.

### CODE REVIEW

**Comment by cshaley** [comment]
@synapticarbors I think this is ready to merge.

File: recipes/nodebook/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://github.com/stitchfix/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```
Comment (synapticarbors) [inline recipes/nodebook/meta.yaml]: Could this be `noarch: python`? 

https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

File: recipes/nodebook/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - jupyter
+    - click
+    - dill
+    - msgpack-python
+    - pandas
+    - pytest-runner
+
+test:
+  imports:
+    - nodebook
```
Comment (synapticarbors) [inline recipes/nodebook/meta.yaml]: should we also test importing `nodebook.ipython`?

File: recipes/nodebook/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    - setuptools
+  run:
+    - python
+    - ipython
+    - jupyter
+    - click
+    - dill
+    - msgpack-python
+    - pandas
+    - pytest-runner
```
Comment (synapticarbors) [inline recipes/nodebook/meta.yaml]: Is `pytest-runner` actually needed as a runtime dependency, or is it just a testing dependency?

File: recipes/nodebook/meta.yaml (hunk 1 above)
Comment (cshaley) [reply recipes/nodebook/meta.yaml]: Yes, thanks, will update

File: recipes/nodebook/meta.yaml (hunk 3 above)
Comment (cshaley) [reply recipes/nodebook/meta.yaml]: It's listed in the `install-requires` packages in `setup.py`.  But yeah, I doubt it is necessary.  Will remove it and see if it still works on my machine.

File: recipes/nodebook/meta.yaml (hunk 2 above)
Comment (cshaley) [reply recipes/nodebook/meta.yaml]: yep, done.

**Pull request merged**